pkg/docker/registry/images.go

```diff
@@ -7,6 +7,8 @@ import (
 func MakeProxiedImageURL(proxyHost string, appSlug string, image string) string {
 	parts := strings.Split(image, "@")
 	if len(parts) == 2 {
+		// we have a digest, but need to also check for a tag
+		parts = strings.Split(parts[0], ":")
```

Comment:
Looks like this discards digest and pulls using tag.  Is that consistent with how the library behaves in other (non-proxied) cases?

Reply:
this method already discards the tag or digest (now both if present) to create the proxied image url to use as `newName` for kustomize